manim/renderer/renderer.py

```diff
@@ -29,34 +31,46 @@ def __init__(self):
             (OpenGLVMobject, self.render_vmobject),
             (ImageMobject, self.render_image),
         ]
+        self._unsupported_mob_warnings: set[str] = set()
 
     def render(self, state: SceneState) -> None:
         self.pre_render(state.camera)
         for mob in state.mobjects:
-            for type_, render_func in self.capabilities:
-                if isinstance(mob, type_):
-                    render_func(mob)
-                    break
-            else:
-                logger.warn(
-                    f"The type{type(mob)} is not supported in Renderer: {self.__class__}"
-                )
+            self.render_mobject(mob)
         self.post_render()
 
+    def render_mobject(self, mob: OpenGLMobject) -> None:
+        for MobClass, render_func in self.capabilities:
+            if isinstance(mob, MobClass):
```

Comment:
```suggestion
        for mob_cls, render_func in self.capabilities:
            if isinstance(mob, mob_cls):
```
python conventions, it's still technically a variable (even though it stores a `type` object)